packages/react-theme/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './themes/index';
 export * from './utils/index';
+export * from './global/brandColors';
```

Comment:
why are only brand colours exported, but not any other colour ?

Reply:
Application should never use global colours as those are not theme-aware. We will never export those.
How brands are different is those are an input to `createXXXTheme()` functions which are exported.

Reply:
Hello, I was just made aware of this PR and I have a very similar one: #21716, I think exporting everything from `globals` would make sense, I think things like [colors]*https://github.com/microsoft/fluentui/blob/master/packages/react-theme/src/global/colors.ts) or [fonts](https://github.com/microsoft/fluentui/blob/master/packages/react-theme/src/global/fonts.ts) could be very widely used.

We'd love to be able to just use these kind of values as part of our UX defaults as part of [office-bohemia](https://office.visualstudio.com/OC/_git/office-bohemia).